src/Editor/GUI/Editors/PlaneShapeEditor.hpp

```diff
@@ -11,6 +11,9 @@ namespace GUI {
     /// Plane shape editor for physics components.
     class PlaneShapeEditor : public IShapeEditor {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The class 'PlaneShapeEditor' does not have a constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AnoConstructor)